web/src/components/ChatMessage.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
+
+import buildConfig from '../constants/buildConfig'
+import RemoteContent from './RemoteContent'
+import { ChatMessageType } from './__mocks__/ChatMessages'
+import Icon from './base/Icon'
+
+const Message = styled.div<{ isAuthor: boolean }>`
```

Comment:
🙃  We've been getting a bunch of warning messages in the browser console from passing in props that styled-components doesn't recognize. We have an open issue for it somewhere anyway and it can be done there just as well but if you touch this again, maybe rename these props to have a `$` in the beginning, then styled-components knows to ignore them. So `$isAuthor` instead of `isAuthor` in this case.

Reply:
Wait, is `isAuthor` going to be used in the future? Because it's not used in `Message` at the moment.

Reply:
i removed it from message and added a `$`for the Container and `IconContainer`